src/dabplus-on-air-processing.liq

```diff
@@ -39,10 +39,20 @@ output =
     input
   end
 
-# send the processed signal to the encoder
+# send the processed signal to the ODR encoder
+odr_encoder =  string.concat(
+  separator=" ", 
+  [
+    encoder_bin, 
+    "--input=stdin",
+    "--output=#{encoder_output}",
```

Comment:
Is it still possibile to use more than one ZMQ output? This is required in order to send a stream to a remote ZMQ instance (aka DAB+ provider) and another one to a local DabMUX for quality checks and debug purposes.